src/openzaak/components/catalogi/utils.py

```diff
@@ -43,6 +43,39 @@ def has_overlapping_objects(
     return False
 
 
+def has_overlapping_objects_with_concepts(
+    model_manager: models.Manager,
+    catalogus: Catalogus,
+    omschrijving_query: dict,
+    begin_geldigheid: date,
+    einde_geldigheid: Optional[date] = None,
+    instance: Optional[models.Model] = None,
+    concept: bool = None,
+) -> bool:
+    if concept:
```

Comment:
I think the function already deserves docstrings

Reply:
Implemented a docstring